wp-content/themes/twentyseventeen-wp20/template-parts/header/site-branding.php

```diff
@@ -1,9 +1,14 @@
 <div class="site-branding">
+
+	<a class="wp20-wordpress-mark" href="https://wordpress.org/">
+		<img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/images/wordpress-mark.svg" alt="<?php esc_html_e( 'WordPress mark', 'wp20' ); ?>">
+	</a>
+
 	<?php the_custom_logo(); ?>
 
-	<?php if ( is_front_page() ): ?>
+	<?php if ( is_page( 'whats-on' ) ): ?>
```

Comment:
I've set things so that the large header only shows on What's on, if we launch with News as the front page initially, it **will not** get the large header. Let me know if you think that's the wrong call.

Reply:
Why does the previous code not work?

Reply:
It did with What's On set to be the front page, but my point is that the front page might be News initially, and we don't want this text displayed in that case, hence the change.

Reply:
I got you. 

Is the copy specifically related to the "what's-on" page or just the homepage itself? I feel like the order of the nav will dictate that, so whether `news` is first of `what's on`, I imagine the front page gets that title. idk.

Reply:
Yeah the more I think about it the more I think this large header and copy should be shown if `is_front_page`, regardless of what page that is... might revert

Reply:
Reverted. Large header now shown on front page, regardless of which page that is.